code/datums/components/footstep.dm

```diff
@@ -31,7 +31,7 @@
 		return
 
 	var/mob/living/LM = parent
-	if(LM.buckled || LM.throwing || LM.is_ventcrawling)
+	if(LM.buckled || LM.throwing || LM.is_ventcrawling || LM.stat == DEAD)
```

Comment:
should this also check for uncon?

Reply:
Not unless/until it's coded to use an alternate sound when dragged and lying down. I just want to stop it making a sound when she's already dead as it's confusing people.